client/jetpack-connect/main.jsx

```diff
@@ -388,17 +143,15 @@ export class JetpackConnectMain extends Component {
 	}
 
 	render() {
-		const status = this.getStatus();
-		const { type } = this.props;
+		const { renderFooter, status, type } = this.props;
 
 		return (
 			<MainWrapper>
 				{ this.renderLocaleSuggestions() }
 				<div className="jetpack-connect__site-url-entry-container">
 					<MainHeader type={ type } />
-
 					{ this.renderSiteInput( status ) }
-					{ this.renderFooter() }
+					{ renderFooter() };
```

Comment:
This is the nitpick-iest of nitpicks, but did you mean to have the trailing `;` here?